larpandoracontent/LArControlFlow/MasterAlgorithm.h

```diff
@@ -307,6 +307,8 @@ class MasterAlgorithm : public pandora::ExternallyConfiguredAlgorithm
 
     bool                        m_workerInstancesInitialized;       ///< Whether all worker instances have been initialized
 
+    unsigned int                m_larCaloHitVersion;		    ///< The LarCaloHit version for LArCaloHitFactory
```

Comment:
Ensure declared in same order as initialisation in any constructors, and assignment in the ReadSettings callback.

Check indentation of comment, and capitalisation of LArCaloHit.

Reply:
Thanks, I've fixed the comment indentation! The capitalisation is the same as for m_larCaloHitFactories.
The variable is declared in same order as constructor initialisation, but the order in ReadSettings is different, as discussed [here](https://github.com/PandoraPFA/LArContent/pull/151/files#r533328123).